assigner.py

```diff
@@ -83,12 +80,14 @@ def autoassign_rras(bapi, cfg, dry_run):
 
     try:
         bugzilla.DotDict(bugs[-1])
-        logger.debug("Found {} unassigned RRA(s). Assigning work!".format(len(bugs)))
+        logger.debug("Found {} unassigned bug(s). Assigning work!".format(len(bugs)))
         for bug in bugs:
             # Is this a valid rra request bug?
-            if bug.get('whiteboard').startswith('autoentry'):
-                logger.debug("{} is not an RRA, skipping".format(bug.get('id')))
-                continue
+
+            # TODO: figure out why this is necessary
+            # if bug.get('whiteboard').startswith('autoentry'):
```

Comment:
@gdestuynder for new RRA bugs, who's setting this whiteboard?  Is it essential or can we drop it?